src/app/my-regular-giving/my-regular-giving.component.html

```diff
@@ -15,16 +15,27 @@
         @if (mandates.length > 0) {
           <biggive-grid columnCount="2">
             @for (mandate of mandates; track mandate.id) {
-              <biggive-basic-card
-                [icon]="false"
-                [textColour]="'primary'"
-                [mainImageUrl]="''"
-                [mainTitle]="mandate.charityName"
-                subtitle="{{(mandate.amount.amountInPence / 100) | exactCurrency:mandate.amount.currency }}, {{ordinal(mandate.schedule.dayOfMonth)}} of each month, {{mandate.giftAid ? 'with Gift Aid, ' : ''}} since {{ mandate.schedule.activeFrom | date: 'mediumDate'}}"
-                [buttonUrl]="'/campaign/' + mandate.campaignId"
-                [buttonAlign]="'center'"
+              <biggive-container-card
+                card-colour="white"
+                background-colour="primary"
               >
-              </biggive-basic-card>
+                <div class="mandate">
+                  <h3><a href="{{'/campaign/' + mandate.campaignId}}">{{ mandate.charityName }}</a></h3>
+                <ul>
+                  <li>Amount: {{(mandate.amount.amountInPence / 100) | exactCurrency:mandate.amount.currency }}</li>
```

Comment:
@DomTBG asked for 2 digits after decimal point, but I'm not sure if we do want that. This is only showing a decimal point if it's not a whole number of pounds (which on this line it probably never will be).  This matches the style we use in [many other places across the site](https://github.com/search?q=repo%3Athebiggive%2Fdonate-frontend%20exactCurrency&type=code), and I don't know if there's any reason to deviate from it.